src/ts/components/dashboard/TransactionList.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react'
+import { connect } from 'react-redux'
+import { withRouter, RouteComponentProps } from 'react-router'
+import { Tab, List, Grid, Icon } from 'semantic-ui-react'
+import { IAppState } from '../../background/store/all'
+import { getTransactions, TransactionType, ITransaction } from '../../background/store/transaction'
+import t from '../../services/i18n'
+
+interface ITransactionListProps extends StateProps, DispatchProps, RouteComponentProps {}
+
+interface ITransactionListState {
+  currentAddress: string
+}
+
+class TransactionList extends React.Component<ITransactionListProps, ITransactionListState> {
+
+  state = {
+    currentAddress: ''
```

Comment:
you should be able to get currentAddress from store. 

![image](https://user-images.githubusercontent.com/37568337/59682053-be3c3d00-9218-11e9-9b4a-3372818ba442.png)